<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter_Time_Form</class>
 <widget class="QWidget" name="Parameter_Time_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Parameter_Time_Form
{
	border-image: url(:/resource/images/common/Transparent_Black_70.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>262</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>216</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="minimumSize">
      <size>
       <width>556</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>556</width>
       <height>115</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBg
{
	border-image: url(:/resource/images/parameter_input/Normal/Background_Empty.png);
}</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>48</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(105, 105, 105);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Second:</string>
      </property>
     </widget>
     <widget class="PushButtonDone" name="pushButton">
      <property name="geometry">
       <rect>
        <x>434</x>
        <y>23</y>
        <width>112</width>
        <height>82</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nexa Light</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	border-image: url(:/resource/images/upload_setting/Done_OVER.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:pressed
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>48</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(105, 105, 105);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Millisecond:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>48</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(105, 105, 105);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Minute:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMin">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>41</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 17pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSec">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>41</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 17pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMsec">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>41</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 17pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>217</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>262</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonDone</class>
   <extends>QPushButton</extends>
   <header>PushButtonDone.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditMin</tabstop>
  <tabstop>lineEditSec</tabstop>
  <tabstop>lineEditMsec</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
